Makefile

```diff
@@ -128,15 +128,12 @@ endif
 
 XGO := $(strip $(if $(XGOOS),GOOS=$(XGOOS)) $(if $(XGOARCH),GOARCH=$(XGOARCH)) $(if $(XHOST_TRIPLE),CC=$(CC_PATH) CXX=$(CXX_PATH)) $(GO))
 
-.DEFAULT_GOAL := all
-.PHONY: all
-all: build test lint
```

Comment:
keep it simple:

- don't use DEFAULT_GOAL; `make` by definition uses the first rule defined as default
- just write `all: build` and be done with it (no need for PHONY either, nor the dependency below -- it's implied by all:build)